tasks/main.yml

```diff
@@ -1,6 +1,23 @@
 # tasks/main.yml
```

Comment:
Rewrite please to use `loop`. `with_*` statements are [deprecated](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#migrating-from-with-x-to-loop)